jwst/skymatch/skymatch.py

```diff
@@ -25,7 +25,7 @@
 log.setLevel(logging.DEBUG)
 
 
-def match(images, skymethod='global+match', match_down=True, subtract=False):
+def skymatch(images, skymethod='global+match', match_down=True, subtract=False, skylist=None):
```

Comment:
Slightly beyond the PR scope, but I went ahead and changed the name of this function because it violates https://docs.astral.sh/ruff/rules/builtin-variable-shadowing/

Reply:
is `match` a reserved word?

Reply:
I guess so, as of 3.10: https://docs.python.org/3/reference/compound_stmts.html#the-match-statement

Reply:
see Melanie's comment below, sounds like we should go ahead and change this.